src/contentmanager.cpp

```diff
@@ -242,14 +244,18 @@ void ContentManager::updateLibrary() {
     url.setPort(CATALOG_PORT);
     url.setPath("/catalog/search");
     url.setQuery(query);
+    
     qInfo() << "Downloading" << url.toString(QUrl::FullyEncoded);
-    m_remoteLibrary = kiwix::Library();
-    kiwix::Manager manager(&m_remoteLibrary);
     try {
-        auto allContent = kiwix::download(url.toString(QUrl::FullyEncoded).toStdString());
-        manager.readOpds(allContent, CATALOG_URL);
+        m_remoteLibrary = kiwix::Library();
```

Comment:
This is an open question here: Should we "reset" the library when we do the request (so here), or when we receive
the response of the request (so in updateRemoteLibrary) ?

Reply:
i don't see big differences between these two options.

Reply:
Rethinking about it, it would prefer to se the reset of the library in `updateRemoteLibrary`.
The need for a clear library is required by the fact that the manager doesn't remove books.
This is an implementation detail and it should be an internal step of `updateRemoteLibrary`.